DigitalLearningSolutions.Data.Tests/Services/PasswordResetServiceTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Data;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Models.Email;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FakeItEasy;
+    using Microsoft.Extensions.Logging;
+    using NUnit.Framework;
+
+    public class PasswordResetServiceTests
+    {
+        private PasswordResetService passwordResetService;
+        private IDbConnection connection;
+        private IUserService userService;
+        private ILogger<PasswordResetService> logger;
+        private IConfigService configService;
+        private IEmailService emailService;
+
+        [SetUp]
+        public void Setup()
+        {
+            userService = A.Fake<IUserService>();
+            logger = A.Fake<ILogger<PasswordResetService>>();
+            configService = A.Fake<IConfigService>();
+            emailService = A.Fake<IEmailService>();
+            connection = ServiceTestHelper.GetDatabaseConnection();
+
+            A.CallTo(() => userService.GetUserByEmailAddress(A<string>._)).Returns(new DelegateUser
+            {
+                Id = 1,
+                FirstName = "Forename",
+                Surname = "Surname",
+                EmailAddress = "recipient@example.com",
+                ResetPasswordId = null
+            });
```

Comment:
Some of the Web.Tests project tests have builders under TestHelpers. Not sure if it's really necessary for one user, but just FYI for the future if you want to pull out some of the model constructions. :)